cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -774,12 +974,12 @@ private KafkaRebalanceStatus buildRebalanceStatus(String sessionID, KafkaRebalan
      * @param rebalanceOptionsBuilder builder for the Cruise Control REST API client options
      * @return a Future with the next {@code KafkaRebalanceStatus} bringing the state
```

Comment:
You need to update the javadoc for all the `on<state>` methods as they now return a `Future<MapAndStatus>` not just the status future.